templates/tpos/tpos.html

```diff
@@ -372,7 +372,9 @@ <h5 class="q-mt-none">
           if (typeof NDEFReader == 'undefined') {
             throw {
               toString: function () {
-                return 'NFC not supported on this device or browser.'
+                this.$q.notify({
```

Comment:
I added this as workaround to not indicate anything if NFC is not supported. The goal is not to send any notification.